src/tests/verification_mock/mod.rs

```diff
@@ -264,6 +265,7 @@ fn test_with_mock_root(test_case: &TestCase) {
         test_case.stapled_ocsp.unwrap_or(&[]),
         std::time::SystemTime::now(),
     );
-    assert_eq!(result.map(|_| ()), test_case.expected_result);
+    let _expected = &test_case.expected_result;
+    assert!(matches!(result.map(|_| ()), _expected));
```

Comment:
This is the first time I've used the `matches!` macro and it's a little peculiar. 

Leaving the expected value in-line produces a syntax error about an unexpected `.`, similarly using `expected` not named as `_expected` raises an unused variable warning even though it clearly is used in the match expression.

This is my clumsy way to make it work. Suggestions for better approaches welcome!

Reply:
Update: this is just plain wrong :sweat_smile: Looking into it.